spec/donation_system/data_structs_for_tests.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module DonationSystem
+  RawRequestData = Struct.new(:amount, :currency, :giftaid, :token,
+                              :name, :email,
+                              :address, :city, :state, :zip, :country)
+  RawPaymentData = Struct.new(:status, :amount)
+  SupporterSObjectFake = Struct.new(:AccountId)
```

Comment:
What does `SupporterSObjectFake` mean?

Reply:
`SObject` is Salesforce terminology for a Salesforce object. I'll rename it.